sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -1623,6 +1624,17 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
     visitFunctionName(ctx, ctx.identifier().asScala.map(_.getText))
   }
 
+  /**
+   * Create a function database (optional) and name pair.
+   */
+  protected def visitFunctionName(ctx: FunctionCallNameContext): FunctionIdentifier = {
```

Comment:
it's not an override, probably `getFunctionIdentifier` is clearer

Reply:
Copy, done in d357a9a.